.github/workflows/build.yml

```diff
@@ -212,7 +213,7 @@ jobs:
           echo "Labels: $LABELS"
           echo "Target Branch: $TARGET_BRANCH"
 
-          if [ "$LABELCHECK" == "" ] && [ "$TARGET_BRANCH" != "beta" ] && [ "$TARGET_BRANCH" != "release" ]; then
+          if [ "$LABELCHECK" == "" ] && [ "$TARGET_BRANCH" != "beta" ] && [ "$TARGET_BRANCH" != "release" ] && [ "$TARGET_BRANCH" != "LTS"* ]; then
```

Comment:
That doesn't seem right. I ran `if [ "LTS2022" != "LTS"* ]; then echo 'hi'; fi` in my shell and it printed 'hi', which it probably shouldn't for your purposes.

Reply:
Bah, seems this is a zsh feature.